### ISSUE TITLE
Problem installing chromatic on Windows

### ISSUE THREAD

**Issue by zashaavery** [zkbt/chromatic#201]
I tried to install chromatic-lightcurves on windows and it gave me this error. I have downloaded the Microsoft Visual C++ already and have even tried running the code in the Developer Command Prompt for VS 2022.

```
error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for batman-package
Failed to build batman-package
ERROR: Could not build wheels for batman-package, which is required to install pyproject.toml-based projects
```

**Comment by zkbt** [comment]
My plan for this is to:
- turn `batman-package` into an optional dependency, and just warn people if they want to use it
- explore `python`-only transit codes, or at least ones that are easier to install
- help @zashaavery get set up with being able to compile C code

**Comment by zkbt** [comment]
It looks like the `numpy`-only version of https://github.com/exoplanet-dev/exoplanet-core might do the trick. @catrionamurray , does that seem reasonable to you as a replacement for `batman`? If so, could you help a little with the translation from the kinds of inputs we provide to `inject_transit` to what we need to provide for `flux = quad_limbdark_light_curve(u1, u2, b, ror)`? I think this is probably closely related to what you've needed to be doing for all your `jax` experiments.

**Comment by zkbt** [comment]
@will-waalkes , do you have experience with this `Microsoft C++ Build Tools` error that @zashaavery 's getting? Any ideas? We installed the `visual-cpp-build-tools` but didn't explore too deeply down into what other things we need to install to be able to compile things via `pip`.

**Comment by will-waalkes** [comment]
> @will-waalkes , do you have experience with this `Microsoft C++ Build Tools` error that @zashaavery 's getting? Any ideas? We installed the `visual-cpp-build-tools` but didn't explore too deeply down into what other things we need to install to be able to compile things via `pip`.

If I remember correctly, I installed the Visual Studio tools and got things to start working but I had to play with which version - this page might help https://stackoverflow.com/questions/40504552/how-to-install-visual-c-build-tools.

I believe my process looked like installing different versions of Visual Studio and then trying to run code until one of the versions worked. My longer-term fix to coding on windows is either (a) download a Linux virtualbox (I can help with that) or there are some browser-based IDEs which might allow you to skip over the pesky windows dependencies, google collab might still be a good option for that? I haven't used it in quite some time.

**Comment by catrionamurray** [comment]
> It looks like the `numpy`-only version of https://github.com/exoplanet-dev/exoplanet-core might do the trick. @catrionamurray , does that seem reasonable to you as a replacement for `batman`? If so, could you help a little with the translation from the kinds of inputs we provide to `inject_transit` to what we need to provide for `flux = quad_limbdark_light_curve(u1, u2, b, ror)`? I think this is probably closely related to what you've needed to be doing for all your `jax` experiments.

Yep I can help with this! Shall I create a new branch + pull request using exoplanet-core instead of batman?

**Comment by catrionamurray** [comment]
Added pull request for *exoplanet-core* method that shouldn't require any special imports like batman!
